performance-test/README.md

```diff
@@ -0,0 +1,23 @@
+## Steps to run an upload test with neqo-client and neqo-server:
+
+1. Build the release version of neqo-client and neqo-server by running
+   `cargo build --release`
+1. Start neqo-server. `./target/release/neqo-server --db ./test-fixture/db`
+1. Start neqo-client and specify parameters to start the upload test.
+   ` ./target/release/neqo-client http://127.0.0.1:4433/  --test upload  --upload-size ${size_in_bytes}`
+
+## To enable log messages for analyzing upload performance
+
+This can be done by setting the `RUST_LOG` environment variable to `neqo_transport=info`.
+For example, the command below starts neqo-client and uploads 8MB of content to the server.
+```
+RUST_LOG=neqo_transport=info ./target/release/neqo-client http://127.0.0.1:4433/ --test upload --upload-size 8388608 &>upload.log
+```
+
+## To visualize the log
+
+We have a simple Python script `neqo_cwnd.py` that can be used to visualize the log.
+To run this script, ensure you have `matplotlib` installed.
+```
+python3 neqo_cwnd.py upload.log
+```
```

Comment:
It would be nice if this were a shell script, rather than instructions.

Reply:
Sure, I can also submit a shell script for this.